Docs/SampleCustom.plist

```diff
@@ -1298,36 +1298,21 @@
 				<string>ForceDisplayRotationInEFI</string>
 			</array>
 		</dict>
-		<key>LegacyEnable</key>
-		<false/>
 		<key>LegacyOverwrite</key>
 		<false/>
 		<key>LegacySchema</key>
 		<dict>
+			<key>36C28AB5-6566-4C50-9EBD-CBB920F83843</key>
```

Comment:
Same here.

Reply:
Sorry - I meant to revert these. Will do. Although see my comments in the docs. In fact it is *far* easier in actual use to leave these as `*`, for two reasons:

 1. You can set test vars of arbitrary names see whether things are working as you expect
   - (Also 1a you can examine the contents of the saved file and see everything that was present to save, not just what you allowed it to save, though this is more for debugging the driver itself)
 2. Exactly what macOS saves as installer vars changes on each version (AFAICT), and so it is possible that an installer will fail just because we only include here what we thought was required, based on a previous installer.

Reply:
I've reverted these, and left some comments about this in the documentation.